File: test/data/energyBalance/crop2ml/Algo/cs/Conductance.cs
double h;
h = Math.Max(10, plantHeight) / 100;
conductance = (wind * Math.Pow(vonKarman, 2)) / (Math.Log((heightWeatherMeasurements - d * h) / (zm * h)) * Math.Log((heightWeatherMeasurements - d * h) / (zh * h)));

File: test/data/energyBalance/crop2ml/Algo/java/Conductance.java
double h;
h = Math.max(10, plantHeight) / 100;
conductance = (wind * Math.pow(vonKarman, 2)) / (Math.log((heightWeatherMeasurements - d * h) / (zm * h)) * Math.log((heightWeatherMeasurements - d * h) / (zh * h)));
